app/models/mail_handler.rb

```diff
@@ -161,8 +161,8 @@ def receive_issue
     end
 
     issue = WorkPackage.new(author: user, project: project)
-    issue.safe_attributes = issue_attributes_from_keywords(issue)
-    issue.safe_attributes = { 'custom_field_values' => custom_field_values_from_keywords(issue) }
+    issue.attributes = issue_attributes_from_keywords(issue)
```

Comment:
@why-el please note that `safe_attributes=` has some more behaviour on [Issues/WorkPackages](https://github.com/opf/openproject/blob/dev/app/models/work_package.rb#L518). I haven't yet looked into whether that will present a problem, but it should be double checked. It e.g. checks whether a new status is allowed. Taking the status as an example, the check within safe attributes is IMO misplaced, a validation would be better (as long as one wants to stick to the model layer), but the check exists now and we have to make sure, that a replacement exists.

Additionally, I am undecided whether we need to wrap the call to `issue_attributes_from_keywords` into some permit logic. We are accepting input from an e-mail here. While we are parsing the mail and thus should be able to control the parameters, we would have to maintain the logic about what is allowed for whom within the `MailHandler` which is not a good place as the logic is also required elsewhere.